pagopa/io-app
# fix: CTA bottom margin inside `RemoveAccountDetailScreen`

## Short description
This PR fixes the CTA bottom margin inside `RemoveAccountDetailScreen` in those devices that don't have the home indicator.

<details><summary>Demo</summary>
<p>

| iPhone SE ❌ | iPhone SE ✅ | iPhone SE ✅ |
|--------|--------|--------|
| <img src="https://github.com/user-attachments/assets/cc1fbbbf-6bcf-43a6-87fc-d62af14bfddd" width="250" /> | <img src="https://github.com/user-attachments/assets/46cccf32-b883-496f-9f9b-e7c410714e4e" width="250" /> | <img src="https://github.com/user-attachments/assets/c0b50de0-4b67-4b92-87a1-73d1799bef9e" width="250" /> |

| iPhone 14 ✅ | iPhone 14 ✅ |
|--------|--------|
| <img src="https://github.com/user-attachments/assets/0c88d4d4-4c59-45f5-bb27-5d06afa48c02" width="250" /> | <img src="https://github.com/user-attachments/assets/0dffdc60-11ff-4fc7-9712-0e585964042d" width="250" /> |

| Motorola Neo 30 ❌ | Motorola Neo 30 ✅ | Motorola Neo 30 ✅ |
|--------|--------|--------|
| <img src="https://github.com/user-attachments/assets/9e2c7da4-d4ee-4a4c-9b79-8a5ad9bbebca" width="250" /> | <img src="https://github.com/user-attachments/assets/6c0c35b3-4982-4c39-a1f3-cb55caadd5dc" width="250" /> | <img src="https://github.com/user-attachments/assets/b69daf9e-4cad-482e-b2ee-83d1f91243f8" width="250" /> |

</p>
</details> 

## List of changes proposed in this pull request
- Add a bottom margin equal to `24`  if the `insets.bottom` from `safe-area-view-context` is `0`.

## How to test
Go to `RemoveAccountDetailScreen` and see the correct CTA bottom margin. Test also the `KeyboardAvoidingView` feature by selecting the last removal reason.
